pkg/cluster/fixetcd.go

```diff
@@ -0,0 +1,500 @@
+package cluster
```

Comment:
If the plan is to expose a geneva actions endpoint instead, should we move all of this to `pkg/frontend` rather than leave it here?

Reply:
If we want to move it to package frontend in this PR, I think we should wait for the endpoint to merge. Lots of code needs to be updated to make use of `KubeActions` in frontend.